Makefile

```
@@ -0,0 +1,54 @@
+#### Variables ####
+
+export ROOT_DIR ?= $(PWD)
+export SRS_ROOT_DIR ?= $(ROOT_DIR)
```

I think everything related to `srs` should be changed to `srsran` such that we keep consistency across the different repos. Given that the repo's name is `aether-sraran`, the "variable" to be used here should be `srsran` as it is done in the other repos inside the `deps` directory.  For example:
```suggestion
export SRSRAN_ROOT_DIR ?= $(ROOT_DIR)
```

---

Done